FAIMS/FAIMS3
# Instantiate survey from template

# feat/fix/chore: pr title

## JIRA Ticket

https://jira.csiro.au/browse/BSS-322


## Description

A "Create New Survey" button is displayed on the list of surveys screen. When clicked, it navigates to the /create-new-survey route.

On this page, the user can select a template from a dropdown menu. Once a template is selected, the "Create Survey" button becomes enabled.

Clicking the button opens a popup where the user is prompted to enter a survey name. After entering the name and submitting, a new survey will be created based on the selected template.

The newly created survey will then appear in the list of available surveys.
## Proposed Changes

List the changes made in this PR.

## How to Test

Open App
On list of surveys page, click 'Create new survey' button
Choose a template from dropdown
Add survey name in the popup and click confirm
New survey created will be displayed in Available/Inactive Tab.


## Checklist
